Signal/src/ViewControllers/MediaPageViewController.swift

```diff
@@ -78,20 +78,31 @@ class MediaPageViewController: UIPageViewController, UIPageViewControllerDataSou
 
         let kSpacingBetweenItems: CGFloat = 20
 
-        let headerView = UIStackView()
-        self.headerView = headerView
+        self.headerView = UIView()
+        headerView.layoutMargins = UIEdgeInsets(top: 2, left: 8, bottom: 4, right: 8)
```

Comment:
I too find myself wrapping `UIStackView` in a UIView with layout margins often.

Reply:
Hmm.... I wouldn't normally do something like this, and instead I'd assign the layout margins to the StackView directly, but the hangup here is that because we're dealing with views in the navbar on iOS9/10 we have to set an explicit frame, so the stack view can't "self size".

I can't think of a reason to do it outside of that, but maybe I'm missing something.